File: app/src/main/java/com/ruslanabzalov/pocketdoc/doctors/SpecialitiesList.java
package com.ruslanabzalov.pocketdoc.doctors;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class SpecialitiesList {

    @SerializedName("SpecList")
    private List<Speciality> mSpecialities;

    public void setSpecialities(List<Speciality> specialities) {
        mSpecialities = specialities;
    }

    public List<Speciality> getSpecialities() {
        return mSpecialities;
    }
}

File: app/src/main/java/com/ruslan/pocketdoc/data/SpecialitiesList.kt
package com.ruslan.pocketdoc.data

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class SpecialitiesList(
        @SerializedName("SpecList")
        @Expose
        val specialitiesList: List<Speciality>
)